plugins/jenkins/README.md

```diff
@@ -84,6 +84,5 @@ YWRtaW46MTFlYzI1NmU0Mzg1MDFjM2Y1Yzc2Yjc1MWE3ZTQ3YWY4Mw== is the base64 of user a
 
 ## Limitations
 
-- Only works with projects that use the Git SCM
-- It requires jobs to be organised into folders
-- No pagination support currently - don't run this on a Jenkins with lots of builds
+- Only works with organization folder jobs backed by GitHub
+- No pagination support currently, limited to 50 jobs - don't run this on a Jenkins with lots of builds
```

Comment:
yup, works for us because we only have global build discarders setup to only keep ~10 builds

Reply:
Generally same at my firm, though does vary (I had one pipeline with a few hundred and the display did take a while).